src/fireedge/src/modules/components/List/NestedLabelTree/Table.js

```diff
@@ -0,0 +1,280 @@
+        formData: data,
+        state: treeState,
+        info: __info,
+      })
+
+      setTree(
+        buildLabelTreeState({ labels: updatedTree, user: { ID: __info?.uId } })
+      )
+      resetInitialState()
+    },
+    [addLabel, treeState, __info]
+  )
+
+  const handleCreateLabel = () => {
+    showModal({
+      id: 'create-label',
+      dialogProps: {
+        title: 'Create Label',
+        dataCy: 'modal-create-label',
+        fixedWidth: '500px',
+        fixedHeight: '500px',
+      },
+      form: Labels.CreateForm,
+      onSubmit: handleSubmit,
+    })
+  }
+
+  const columns = [
+    {
+      field: 'name',
+      headerName: Tr(T.Label),
+      flex: 1,
+      minWidth: 120,
+      headerAlign: 'left',
+      align: 'left',
+      disableColumnMenu: true,
+      renderCell: (params) => <Box sx={{ pl: 1 }}>{params.value}</Box>,
+    },
+    {
+      field: 'displayPath',
+      headerName: Tr(T.FullPath),
+      flex: 3,
+      minWidth: 200,
+      headerAlign: 'left',
+      align: 'left',
+      disableColumnMenu: true,
+      renderCell: (params) => <Box sx={{ pl: 1 }}>{params.value}</Box>,
+    },
+    {
+      field: 'actions',
+      headerName: Tr(T.Actions),
+      minWidth: 120,
+      sortable: false,
+      filterable: false,
+      disableColumnMenu: true,
+      renderCell: (params) => (
+        <Box
+          sx={{
+            width: '100%',
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center',
+          }}
+        >
+          <ButtonToTriggerForm
+            buttonProps={{
+              importance: STYLE_BUTTONS.IMPORTANCE.SECONDARY,
+              size: STYLE_BUTTONS.SIZE.MEDIUM,
+              type: STYLE_BUTTONS.TYPE.NOBORDER,
+              icon: <Trash />,
+              'data-cy': `remove-label-${params.row.displayPath}`,
+              tooltip: T.Remove,
+            }}
+            options={[
+              {
+                isConfirmDialog: true,
+                dialogProps: {
+                  children: (
+                    <div style={{ padding: '8px' }}>
+                      <h2
+                        style={{
+                          fontSize: '18px',
+                          fontWeight: 'bold',
+                          marginBottom: '12px',
+                        }}
+                      >
+                        {T.DeleteLabelConcept}
+                      </h2>
+                      <p style={{ marginBottom: '8px' }}>
+                        {T.DeleteTheFollowingLabel}:
+                      </p>
+                      <div style={{ paddingLeft: '12px' }}>
+                        <div style={{ marginBottom: '6px' }}>
+                          <span style={{ fontWeight: 'bold' }}>Name: </span>
+                          <span>{params.row.name}</span>
+                        </div>
+                        <div>
+                          <span style={{ fontWeight: 'bold' }}>Path: </span>
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

JSX element content is not internationalized. Application should support multiple languages through internationalization. Instead, utilize packages like &#39;i18next&#39; to internationalize your items.